conf/services.conf

```diff
@@ -81,8 +81,7 @@ simple_api_endpoints = {
             PhysicalDescription = ^ca_objects.PhysicalDescription,
             Subjects = "["<unit relativeTo="ca_occurrences" delimiter="," restrictToTypes="subject">&quot;^preferred_labels&quot;</unit>"]",
             Summary = ^ca_objects.Summary,
-            Notes = ^ca_objects.Notes,
-            TermsOfUse = ^ca_objects.TermsOfUse
```

Comment:
Ah but the one under notes on page 37 does have one ;). The second one is the terms of use on the item donor relationship I believe. Anyway this rung a bell and according to https://youtrack.gaiaresources.com.au/youtrack/issue/RWAHS-296, this is internal use only, so it should come out.